resources/datomic/schema.edn

```diff
@@ -1,16 +1,48 @@
 [;; User entities
  {:db/id          #db/id [:db.part/db]
-  :db/ident       :user/username
+  :db/ident       :user/id
+  :db/valueType   :db.type/string
+  :db/unique      :db.unique/identity
+  :db/cardinality :db.cardinality/one
+  :db/doc         "A user's public id"}
+
+ {:db/id          #db/id [:db.part/db]
+  :db/ident       :user/handle
   :db/valueType   :db.type/string
   :db/cardinality :db.cardinality/one
-  :db/doc         "A user's username"}
+  :db/doc         "A user's wombat handle"}
 
  {:db/id          #db/id [:db.part/db]
-  :db/ident       :user/github-name
+  :db/ident       :user/access-token
   :db/valueType   :db.type/string
   :db/unique      :db.unique/identity
   :db/cardinality :db.cardinality/one
-  :db/doc         "Users GitHub username."}
+  :db/doc         "A user's GitHub auth token."}
+
+ {:db/id          #db/id [:db.part/db]
+  :db/ident       :user/github-username
+  :db/valueType   :db.type/string
+  :db/cardinality :db.cardinality/one
+  :db/doc         "A user's GitHub username."}
+
+ {:db/id          #db/id [:db.part/db]
+  :db/ident       :user/github-id
+  :db/valueType   :db.type/long
+  :db/cardinality :db.cardinality/one
+  :db/doc         "A user's GitHub id."}
+
+ {:db/id          #db/id [:db.part/db]
+  :db/ident       :user/email
+  :db/valueType   :db.type/string
+  :db/unique      :db.unique/identity
+  :db/cardinality :db.cardinality/one
+  :db/doc         "A user's email from their GitHub account."}
+
+ {:db/id          #db/id [:db.part/db]
+  :db/ident       :user/avatar-url
+  :db/valueType   :db.type/string
+  :db/cardinality :db.cardinality/one
+  :db/doc         "A user's avatar url from their GitHub account."}
```

Comment:
So this is set every time a user logs in (along with their auth token and other values that come back from the github request). It would but more effort on the client to have to make that request for themselves, but they also require a way to fetch all other user profile pics (if that is a feature we need to support).